kolibri/plugins/user_auth/assets/src/views/LogoutSideNavEntry.js

```diff
@@ -0,0 +1,30 @@
+import { UserKinds, NavComponentSections } from 'kolibri.coreVue.vuex.constants';
```

Comment:
The logout side nav entry is defined in core: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/LogoutSideNavEntry.vue

Probably  best that it stay here (as it's not tied to any plugin).

Worth noting that when you convert it to a config object rather than a component, you would need to make sure the translator namespace is identical to the component `name`: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/LogoutSideNavEntry.vue